cpp/mrc/include/mrc/edge/edge_builder.hpp

```diff
@@ -176,6 +176,89 @@ struct EdgeBuilder final
         sink.set_readable_edge_handle(edge);
     }
 
+    template <typename EdgeDataTypeT, typename SourceT, typename SinkT, typename SpliceInputT, typename SpliceOutputT>
+    static void make_edge_splice(SourceT& source, SinkT& sink, SpliceInputT& splice_input, SpliceOutputT& splice_output)
```

Comment:
Why `make_edge_splice` instead of a more active name like `splice_edge`?

Reply:
I actually like that terminology better, will update.